.github/workflows/deploy-pr.yml

```diff
@@ -0,0 +1,73 @@
+name: IssueOps Deploy PR
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  deploy:
+    if: ${{ github.event.issue.pull_request }}
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    steps:
+      - uses: github/branch-deploy@v5.2.1
+        id: branch-deploy
+        with:
+          skip_ci: 'production'
+          trigger: '.deploy' # the trigger phrase to look for in the comment on the pull request
```

Comment:
Just make this deploy whenever pr is created; not like we pay for actions or anything. Also make sure it updates without needing to comment again

Reply:
if we make it automatic, we need to add a way to exclude PR based on something in description, like .exclude

Reply:
I think excluding drafts might be reasonable, e.g. `if: github.event.pull_request.draft == false`